app/config/drupal9-demo/install.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+## install.sh -- Create config files and databases; fill the databases
+[ -d "$WEB_ROOT/web" ] && CMS_ROOT="$WEB_ROOT/web"
+
+###############################################################################
+## Create virtual-host and databases
+
+amp_install
+
+###############################################################################
+## Setup Drupal (config files, database tables)
+
+drupal8_install
+DRUPAL_SITE_DIR=$(_drupal_multisite_dir "$CMS_URL" "$SITE_ID")
+pushd "${CMS_ROOT}/sites/${DRUPAL_SITE_DIR}" >> /dev/null
+  drush8 -y updatedb
+popd >> /dev/null
+
+###############################################################################
+## Setup CiviCRM (config files, database tables)
+
+CIVI_DOMAIN_NAME="Demonstrators Anonymous"
+CIVI_DOMAIN_EMAIL="\"Demonstrators Anonymous\" <info@example.org>"
+CIVI_CORE="${CMS_ROOT}/vendor/civicrm/civicrm-core"
+CIVI_UF="Drupal8"
```

Comment:
6 => Drupal6
7 => Drupal
8 => Drupal8
9 => Drupal8

I get it but ... yeah, drupal.

Reply:
Yeh d8/9 are very similar so no real reason creating a separate UF for it